sms_939/models/__init__.py

```diff
@@ -13,4 +13,5 @@
 from . import sms_notification
 from . import compassion_child
 from . import recurring_contract
```

Comment:
In the model recurring_contract you should inherit the method `associate_group` and call the method `on_change_payment_mode` from there.